File: engine/src/main/java/client/database/data/PlayerNpcFieldData.java
package client.database.data;

public class PlayerNpcFieldData {
   private int worldId;

   private int mapId;

   private int step;

   private int podium;

   public PlayerNpcFieldData(int worldId, int mapId, int step, int podium) {
      this.worldId = worldId;
      this.mapId = mapId;
      this.step = step;
      this.podium = podium;
   }

   public int getWorldId() {
      return worldId;
   }

   public int getMapId() {
      return mapId;
   }

   public int getStep() {
      return step;
   }

   public int getPodium() {
      return podium;
   }
}

File: engine/engine-model/src/main/scala/client/database/data/PlayerNpcFieldData.scala
package client.database.data

class PlayerNpcFieldData(private var _worldId: Int, private var _mapId: Int, private var _step: Int,
                         private var _podium: Int) {
  def worldId: Int = _worldId

  def mapId: Int = _mapId

  def step: Int = _step

  def podium: Int = _podium
}
